components/elm/src/biogeochem/DryDepVelocity.F90

```diff
@@ -275,24 +270,31 @@ subroutine depvel_compute( bounds, &
-            if (elmveg == nbrdlf_dcd_trp_tree                 ) wesveg = 4
-            if (elmveg == nbrdlf_dcd_tmp_tree                 ) wesveg = 4
-            if (elmveg == nbrdlf_dcd_brl_tree                 ) wesveg = 4
-            if (elmveg == nbrdlf_evr_shrub                    ) wesveg = 11
-            if (elmveg == nbrdlf_dcd_tmp_shrub                ) wesveg = 11
-            if (elmveg == nbrdlf_dcd_brl_shrub                ) wesveg = 11
-            if (elmveg == nc3_arctic_grass                    ) wesveg = 3
-            if (elmveg == nc3_nonarctic_grass                 ) wesveg = 3
-            if (elmveg == nc4_grass                           ) wesveg = 3
-            if (elmveg == nc3crop                             ) wesveg = 2
-            if (elmveg == nc3irrig                            ) wesveg = 2
-            if (elmveg >= npcropmin .and. elmveg <= npcropmax ) wesveg = 2
+            if (elmveg == noveg                                      ) wesveg = 8
+            if (nonvascular(elmveg) == 2.0_r8                        ) wesveg = 8      ! assuming lichen like bare-ground
+            !if (elmveg == ndllf_evr_tmp_tree                  ) wesveg = 5
+            !if (elmveg == ndllf_evr_brl_tree                  ) wesveg = 5
+            !if (elmveg == ndllf_dcd_brl_tree                  ) wesveg = 5
+            if (woody(elmveg) == 1.0_r8 .and. needleleaf(elmveg) == 1) wesveg = 5
+            !if (elmveg == nbrdlf_evr_trp_tree                 ) wesveg = 4
+            !if (elmveg == nbrdlf_evr_tmp_tree                 ) wesveg = 4
+            !if (elmveg == nbrdlf_dcd_trp_tree                 ) wesveg = 4
+            !if (elmveg == nbrdlf_dcd_tmp_tree                 ) wesveg = 4
+            !if (elmveg == nbrdlf_dcd_brl_tree                 ) wesveg = 4
+            if (woody(elmveg) == 1.0_r8 .and. needleleaf(elmveg) == 0) wesveg = 4
+            !if (elmveg == nbrdlf_evr_shrub                    ) wesveg = 11
+            !if (elmveg == nbrdlf_dcd_tmp_shrub                ) wesveg = 11
+            !if (elmveg == nbrdlf_dcd_brl_shrub                ) wesveg = 11
+            if (woody(elmveg) == 2.0_r8                              ) wesveg = 11
+            !if (elmveg == nc3_arctic_grass                    ) wesveg = 3
+            !if (elmveg == nc3_nonarctic_grass                 ) wesveg = 3
+            !if (elmveg == nc4_grass                           ) wesveg = 3
+            if (graminoid(elmveg) == 1.0_r8                          ) wesveg = 3
+            if (nonvascular(elmveg) == 1.0_r8                        ) wesveg = 3      ! assuming moss like grass
+            !if (elmveg == nc3crop                             ) wesveg = 2
+            !if (elmveg == nc3irrig                            ) wesveg = 2
+            !if (elmveg >= npcropmin .and. elmveg <= npcropmax ) wesveg = 2
+            if (crop(elmveg) == 1.0_r8 .or. iscft(elmveg) == 1.0_r8  ) wesveg = 2
```

Comment:
Here and a few other places iscft ==1 but almost everywhere else, the use is iscft>=1? I think it would be good to be consistent with the notation.

Reply:
Thanks -based on this and Ryan's comments I converted the iscft to a logical type in the latest commit.